<?xml version="1.0" encoding="UTF-8"?>
<!--
 Copyright (C) 2018 Dzmitry Hatouka (Gotowka). htotatut@gmail.com

 This program is free software; you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation; either version 3, or (at your option)
 any later version.

 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.

 You should have received a copy of the GNU General Public License
 along with this program. See COPYING.txt
 If not, see <https://www.gnu.org/licenses/>.

 Please note that this program has CLA (Contributor Licence Agreement).
 You accept it by making any contribution to this program.
 See file CONTRIBUTE.txt for more details.
 List of 3-rd party components and their licenses is in 3RDPARTIES.txt
-->
<ui version="4.0">
 <class>AddPersonDialog</class>
 <widget class="QDialog" name="AddPersonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a new person</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Choose Build 36 Raw Data Concatenated (GZIP, CSV) file from FTDNA.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listOutput">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>531</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="chbRawData">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>83</y>
     <width>431</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Add person without raw data (no admixture information available)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
